src/_pytest/pytester.py

```diff
@@ -1488,8 +1503,33 @@ def runpytest_subprocess(
         plugins = [x for x in self.plugins if isinstance(x, str)]
         if plugins:
             args = ("-p", plugins[0], *args)
-        args = self._getpytestargs() + args
-        return self.run(*args, timeout=timeout)
+
+        env = os.environ.copy()
+        pythonpath = env.get("PYTHONPATH", "")
+
+        if hasattr(self, "_syspath_prepended"):
+            pythonpath = os.pathsep.join(
+                filter(None, [self._syspath_prepended, pythonpath])
+            )
+
+        env["PYTHONPATH"] = pythonpath
+
+        python_executable = sys.executable
+        pytest_command = [python_executable, "-m", "pytest"]
+
+        if hasattr(self, "_syspath_prepended"):
```

Comment:
Why are we setting `sys.path` *and* configuring `PYTHONPATH`? Isn't sufficient to only configure `PYTHONPATH`?

Reply:
Actually I don't think we need to modify `runpytest_subprocess` at all, given we are already configuring `PYTHONPATH` in `popen`?